package.json

```diff
@@ -6,43 +6,61 @@
     "Dan Gebhardt <dan@cerebris.com>",
     "Tom Dale <tom@tomdale.net>"
   ],
-  "scripts": {},
   "repository": "https://github.com/glimmerjs/glimmer-application-pipeline",
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "./lib/broccoli/glimmer-app.js",
   "typings": "dist/index.d.ts",
   "dependencies": {
     "@glimmer/compiler": "^0.23.0-alpha.2",
     "@glimmer/di": "^0.1.8",
     "@glimmer/resolution-map-builder": "^0.1.0",
+    "amd-name-resolver": "^0.0.6",
+    "babel-plugin-check-es2015-constants": "^6.22.0",
     "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-minify-numeric-literals": "^0.0.1",
+    "babel-plugin-transform-es2015-arrow-functions": "^6.22.0",
+    "babel-plugin-transform-es2015-block-scoping": "^6.23.0",
+    "babel-plugin-transform-es2015-classes": "^6.23.0",
+    "babel-plugin-transform-es2015-computed-properties": "^6.22.0",
+    "babel-plugin-transform-es2015-destructuring": "^6.23.0",
+    "babel-plugin-transform-es2015-for-of": "^6.23.0",
+    "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
+    "babel-plugin-transform-es2015-parameters": "^6.23.0",
+    "babel-plugin-transform-es2015-shorthand-properties": "^6.22.0",
+    "babel-plugin-transform-es2015-spread": "^6.22.0",
+    "babel-plugin-transform-es2015-template-literals": "^6.22.0",
+    "babel-plugin-transform-proto-to-assign": "^6.23.0",
```

Comment:
if we add the plugins explicitly should we still depend on `babel-preset-es2015`?